lib/ui/playground/data/playground_tasks_datasource.dart

```diff
@@ -246,30 +242,39 @@ class PlayGroundTasksDataSource implements PlaygroundDataSource {
           ],
         ),
       ),
-       PlaygroundWidget(
-          title: 'TASK FARM-97 Update Recomentations',
-          child: PlaygroundView(widgetList: [
-            PlaygroundWidget(
-                title: "TASK FARM-97 View List states",
-                child: RecommendationsList(
-                    provider: SequencedViewModelProvider<
-                        RecommendationsListViewModel>(
-                  [
-                    MockRecommendationsListViewModel().build(),
-                    MockRecommendationsListViewModel().build(),
-                    MockRecommendationsListViewModel().build(),
-                    MockRecommendationsListViewModel().build(),
-                  ],
-                ))),
-            PlaygroundWidget(
-                title: "TASK FARM-97 Mock repo",
-                child: RecommendationsList(
-                    provider: RecommendationListProvider(
-                  title: "Mock Repo",
-                  cropRepo: _mockCrop,
-                  plotRepo: _mockPlot,
-                  engine: _engine,
-                ))),]))
+      PlaygroundWidget(
+        title: 'TASK FARM-97 Update Recomentations',
+        child: PlaygroundView(widgetList: [
+          PlaygroundWidget(
+              title: "TASK FARM-97 View List states",
+              child: RecommendationsList(
+                  provider: SequencedViewModelProvider<
+                      RecommendationsListViewModel>(
```

Comment:
Can we move the `<` to the next line?

Reply:
DONE. 
Added missing commas to format it correctly